src/super_gradients/training/models/conversion.py

```diff
@@ -80,6 +86,8 @@ def convert_to_onnx(
     complete_model = ConvertableCompletePipelineModel(model, pre_process, post_process, **prep_model_for_conversion_kwargs)
 
     torch.onnx.export(model=complete_model, args=onnx_input, f=out_path, **torch_onnx_export_kwargs)
+    if simplify:
```

Comment:
Function returns the `out_path` regardless of value of `simplify`.
As a user, if I pass `simplify=True`, then my expectation that result of function return value would be path to simplified model. 

IMHO, we should not generate `_simplified.onnx` at all and produce only one ONNX file. What do you think?

Reply:
Sure, fixed.